Update README for .serve and update language (#11171)
PrefectHQ/prefect: README.md

# Prefect

> Prefect is a workflow application framework. It's the simplest way to deploy Python code as an interactive workflow application that can be observed and orchestrated.

Reviewer: I would not go so hard on "workflow application framework".
```suggestion
Prefect is a developer platform for building, observing, and triaging workflows. Powered by a Python library, it pairs with a responsive dashboard and observability for getting a complete picture of your workflows.
```

Reviewer 2: While I'm comfortable emphasizing the pythonic nature of the SDK, I don't think "Powered by a Python library" is particularly accurate or informative here.

> Prefect is a workflow application framework. It's the simplest way to transform Python code into an interactive workflow application that can be observed and orchestrated.

Reviewer 2: Thats probably a good idea

> Prefect is a workflow application framework. It's the simplest way to transform Python code into an interactive workflow application that can be observed and orchestrated.

Reviewer 3: @discdiver  How about "Prefect is an orchestration and observability platform for building, observing, and triaging workflows. It's the simplest way to transform Python code into an interactive workflow application."

Prefect sits at the center of the many systems you interact with every day.

> It provides you an API that makes it possible to understand and react to events.

Reviewer 2: I see how the API supports reacting to events, but I don't see how it supports understanding events.

> It provides you an API that makes it possible to understand and react to events.

Reviewer: ```suggestion
Prefect allows you to expose your workflows through an API so teams dependent on you can programmatically access your pipelines, business logic, and more.
```

> It provides you an API middleware that makes it possible to understand and react to event.

Reviewer 2: The use of the term "middleware" feels a bit off to me. I think we may need to elaborate on how workflows can act as middleware it to make it clear what we mean. That topic could merit an entire blog post.

> Prefect also allows you to standardize your workflows across your organization.

Reviewer 2: ```suggestion
Prefect also allows you to standardize workflows across your organization.
```

Reviewer: ```suggestion
Prefect also allows you to standardize workflow development and deployment across your organization.
```

> With Prefect, you can build resilient, dynamic workflows that react to the world around them and recover from unexpected changes.

Reviewer 3: I really like this @discdiver

With just a few decorators, Prefect supercharges your code with features like automatic retries, distributed execution, scheduling, caching, and much more.

> Every activity is tracked and can be monitored with a self-hosted [Prefect server](https://docs.prefect.io/latest/guides/host/) instance or managed [Prefect Cloud](https://docs.prefect.io/latest/cloud/) dashboard.

Reviewer: ```suggestion
Every activity is tracked and can be monitored with a self-hosted [Prefect server](https://docs.prefect.io/latest/guides/host/) instance or managed [Prefect Cloud](https://www.prefect.io/cloud-vs-oss?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none) dashboard.
```

# Prefect

Prefect is an orchestration and observability platform for building, observing, and triaging workflows. 
It's the simplest way to transform Python code into an interactive workflow application.

> Prefect sits at the center of the many systems you interact with every day.

Reviewer 2: ```suggestion
```

Reviewer 2: I don't think this is true.

# run the flow!
if __name__=="__main__":
    github_stars(["PrefectHQ/Prefect"])
```

Fire up the Prefect UI to see what happened:

```bash
prefect server start
```

![Prefect UI dashboard](/docs/img/ui/cloud-dashboard.png)

To run your workflow on a schedule, turn it into a deployment and schedule it to run every minute by changing the last line of your script to the following:

```python
    github_stars.serve(name="first-deployment", cron="* * * * *")
```

You now have a server running locally that is looking for scheduled deployments!

> Additionally you can run your workflow manually from the UI or CLI - and if you're using Prefect Cloud, you can even run deployments in response to an [events](https://docs.prefect.io/latest/concepts/automations/).

Reviewer 2: ```suggestion
Additionally you can run your workflow manually from the UI or CLI - and if you're using Prefect Cloud, you can even run deployments in response to [events](https://docs.prefect.io/latest/concepts/automations/).
```

## Prefect Cloud

Stop worrying about your workflows.
Prefect Cloud allows you to centrally deploy, monitor, and manage the data workflows you support. With managed orchestration, automations, and webhooks, all backed by enterprise-class security, build production-ready code quickly and reliably.

Read more about Prefect Cloud [here](https://www.prefect.io/cloud-vs-oss?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none) or sign up to [try it for yourself](https://app.prefect.cloud?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none).

![Prefect Automations](/docs/img/ui/automations.png)

## Next steps

There's lots more you can do to orchestrate and observe your workflows with Prefect!

> Start with our [friendly tutorial](https://docs.prefect.io/tutorials/first-steps) or explore the [core concepts of Prefect workflows](https://docs.prefect.io/concepts/).

Author: ```suggestion
Start with our [friendly tutorial](https://docs.prefect.io/tutorials) or explore the [core concepts of Prefect workflows](https://docs.prefect.io/concepts/).
```

## Join the community

> Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow application system. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).

Reviewer: ```suggestion
Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow system, inspired by application development. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).
```

Additionally you can run your workflow manually from the UI or CLI - and if you're using Prefect Cloud, you can even run deployments in response to [events](https://docs.prefect.io/latest/concepts/automations/).

## Prefect Cloud

Stop worrying about your workflows.
Prefect Cloud allows you to centrally deploy, monitor, and manage the data workflows you support. With managed orchestration, automations, and webhooks, all backed by enterprise-class security, build production-ready code quickly and reliably.

Read more about Prefect Cloud [here](https://www.prefect.io/cloud-vs-oss?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none) or sign up to [try it for yourself](https://app.prefect.cloud?utm_source=oss&utm_medium=oss&utm_campaign=oss_gh_repo&utm_term=none&utm_content=none).

![Prefect Automations](/docs/img/ui/automations.png)

## Next steps

There's lots more you can do to orchestrate and observe your workflows with Prefect!
Start with our [friendly tutorial](https://docs.prefect.io/tutorials) or explore the [core concepts of Prefect workflows](https://docs.prefect.io/concepts/).

## Join the community

> Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow system, inspired by application development. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).

Reviewer 2: ```suggestion
Prefect is made possible by the fastest growing community of thousands of friendly data engineers. Join us in building a new kind of workflow system. The [Prefect Slack community](https://prefect.io/slack) is a fantastic place to learn more about Prefect, ask questions, or get help with workflow design. All community forums, including code contributions, issue discussions, and slack messages are subject to our [Code of Conduct](https://discourse.prefect.io/faq).
```

Reviewer 2: I wouldn't emphasize this but that's a weakly held preference.
